apps/web-giddh/src/app/shared/template-froala/template-froala.component.ts

```diff
@@ -0,0 +1,351 @@
+import { Component, ElementRef, Inject, OnInit, ViewChild } from '@angular/core';
+import { FormBuilder, FormGroup } from '@angular/forms';
+import FroalaEditor from 'froala-editor';
+import { ReplaySubject, takeUntil } from 'rxjs';
+import Tribute from 'tributejs';
+import { CustomEmailComponentStore } from './utility/template-froala.store';
+import { MAT_DIALOG_DATA, MatDialog } from '@angular/material/dialog';
+import 'froala-editor/js/plugins.pkgd.min.js';
+import 'froala-editor/js/froala_editor.pkgd.min.js';
+import { GeneralService } from '../../services/general.service';
+@Component({
+    selector: 'template-froala',
+    templateUrl: './template-froala.component.html',
+    styleUrls: ['./template-froala.component.scss'],
+    providers: [CustomEmailComponentStore]
+})
+export class TemplateFroalaComponent implements OnInit {
+    /** Instance of subject input field */
+    @ViewChild('subjectInputField', { static: false }) subjectInputField: ElementRef;
+    /* Aside pane state*/
```

Comment:
```suggestion
    /** Aside pane state */
```

Use /** at every variable documentation and format also.

Reply:
DONE